packages/apps/plugins/plugin-presenter/src/components/MarkdownSlideMain.tsx

```diff
@@ -4,12 +4,15 @@
 
 import React, { type FC } from 'react';
 
+import { type Document as DocumentType } from '@braneframe/types';
+
 import { Container, Slide } from './Markdown';
 
-export const MarkdownSlideMain: FC<{ slide: any }> = ({ slide }) => {
+export const MarkdownSlideMain: FC<{ document: DocumentType }> = ({ document }) => {
   return (
     <Container>
-      <Slide content={String(slide.content)} />;
+      {/* TODO: content is a YText object, but should be a string. Update after automerge migration. */}
+      <Slide content={document.content.content as any} />;
```

Comment:
@dmaretskyi ended up having to typecast this YText object to any in order to match types. I assume this will all change very shortly here. I've tested this and it works with Automerge documents.